### Probability, Odds, and  Odds ratios

**Probability:** the number of ways that an event can occur divided by the total number of possible outcomes. 

The probability of drawing a red card from a standard deck of cards is 26/52 (50 percent). 
The probability of drawing a club from that deck is 13/52 (25 percent)

#### What's the probability of getting heads in a fair coin flip? 

In [4]:
print "{}%".format(1/float(2) * 100)
# P(X = H) = (thotal Events for Heads)/ Total Events

# Flip one Time X = [H,T]

50.0%


The odds for an event is the ratio of the number of ways the event can occur compared to the number of ways it does not occur.

For example, using the same events as above, the odds for:
drawing a red card from a standard deck of cards is 1:1; and
drawing a club from that deck is 1:3.

#### What's the odds of a fair coin flip?

1:1


#### Suppose that 18 out of 20 patients in an experiment lost weight while using diet A, while 16 out of 20  lost weight using diet B.  

#### What's the probability of weight loss with diet A? What's the odds?

In [5]:
print "{}%".format(18/float(20) * 100)
# prob 

90.0%


9:1

In [9]:
OddsA = float(18)/2
print OddsA

9.0


#### What's the probablity of weight loss with diet B? What's the odds?

In [6]:
print "{}%".format(16/float(20) * 100)

80.0%


In [10]:
OddsB = float(16)/4
print OddsB

4.0


4:1

#### What's the odds ratio?

In [13]:
OddsRatio = OddsA / OddsB
print OddsRatio

2.25


2.25 more likely to loose weight with Diet A than Diet B